romancal/datamodels/schemas/core.schema.yaml

```diff
@@ -38,27 +45,114 @@ properties:
             title: Number of groups in integration
             type: integer
           nframes:
             title: Number of frames per group
             type: integer
-          groupgap:
-            title: Number of frames dropped between groups
-            type: integer
-          frame_time:
-            title: "[s] Time between frames"
-            type: number
-          exposure_time:
-            title: "[s] Effective exposure time"
-            type: number
+          data_problem:
+            title: Science telemetry indicated a problem
+            type: boolean
           gain_factor:
             title: Gain scale factor
             type: number
+          integration_time:
+            title: "[s] Effective integration time"
+            type: number
+          elapsed_exposure_time:
+            title: "[s] Total elapsed exposure time"
+            type: number
+#          nints:
+#            title: Number of integrations in exposure
+#            type: integer
+#          integration_start:
+#            title: Starting integration number in this segment
+#            type: integer
+#          integration_end:
+#            title: Ending integration number in this segment
+#            type: integer
+#          frame_divisor:
+#            title: Divisor applied to frame-averaged groups
+#            type: integer
+#          groupgap:
+#            title: Number of frames dropped between groups
+#            type: integer
+#          drop_frames1:
+#            title: Frames dropped at start of each integration
+#            type: integer
+#          drop_frames3:
+#            title: Frames dropped at end of each integration
+#            type: integer
+#          nsamples:
+#            title: Number of A/D samples per pixel
+#            type: integer
+#          sample_time:
+#            title: "[us] Time between samples"
+#            type: number
+#          frame_time:
+#            title: "[s] Time between frames"
+#            type: number
+#          group_time:
+#            title: "[s] Time between groups"
+#            type: number
+#          exposure_time:
+#            title: "[s] Effective exposure time"
+#            type: number
+#          effective_exposure_time:
+#            title: "[s] Effective exposure time"
+#            type: number
+#          duration:
+#            title: "[s] Total duration of exposure"
+#            type: number
+#          nresets_at_start:
+#            title: Number of resets at start of exposure
+#            type: integer
+#          datamode:
+#            title: post-processing method used in FPAP
+#            type: integer
+#          clock_adjust_rate:
+#            title: "[ms/s] Spacecraft Clock Time Adjust RATE"
+#            type: number
+#          psf_reference:
+#            title: exposure is PSF reference
+#            type: boolean
+#          self_reference_survey:
+#            title: self-referencing PSF
+#            type: boolean
```

Comment:
The MA table comment above applies to here are well.

Reply:
These keywords have been removed.